### ISSUE TITLE
Rendering without a normal controller flow

### ISSUE THREAD

**Issue by drapergeek** [thoughtbot/brady#8]
If you try to render a template directly without going through a controller, brady blows up.

```elixir
# some_plug.ex
conn
|> Phoenix.Controller.put_view(MyApp.ErrorView)
|> Phoenix.Controller.render("test.html")
```

The error you get:
```elixir
key :phoenix_controller not found
```

**Comment by drapergeek** [comment]
I would say that in this case I don't think we'd want to output anything since it's an abnormal situation but I'm open to other ideas.

**Comment by anellis** [comment]
Fixed in commit 708dbb04cc825bc7a3b2abda21d062884336febf
